singleheader/simdjson.h.in

```diff
@@ -409,10 +409,12 @@ constexpr size_t DEFAULT_MAX_DEPTH = 1024;
      * the regular visual studio or clang under visual
```

Comment:
I do not think we want to move the file. We want to keep the file as simdjson.h and we want to make sure that people who drop simdjson.cpp and simdjson.h in their project can make it work transparently.